apps/vscode-wing/src/bin-helper.ts

```diff
@@ -123,3 +123,13 @@ export async function getWingBinAndArgs(context: ExtensionContext) {
 
   return [wingBin, ...args];
 }
+
+export const importCreateConsoleApp = () => {
+  const packagePath = execSync("wing console-app-path").toString().trim();
+
+  const createConsoleApp =
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
+    require(packagePath) as typeof import("@wingconsole/app");
```

Comment:
Maybe do the same for `@wingconsole/server` if it's really needed?

Reply:
not used anymore. removed 👍